core/src/wallet/common/synchronizers/AbstractBlockchainExplorerAccountSynchronizer.h

```diff
@@ -84,22 +85,28 @@ namespace ledger {
             }
         };
 
+        struct BlockchainExplorerAccountSynchronizationResult {
+            Option<uint32_t> reorgBlockHeight;
+            uint32_t lastBlockHeight;
+            uint32_t newOperations;
+        };
```

Comment:
Ideally we would like to have the list of removed operations during sync here inside this result payload **or** by Event Bus means